package.json

```diff
@@ -36,18 +30,17 @@
     "imports-loader": "^0.6.5",
     "iso-3166-2": "0.4.0",
     "isomorphic-fetch": "2.2.1",
-    "istanbul": "1.0.0-alpha.2",
     "jquery": "^2.2.0",
     "jsdom": "8.5.0",
     "jsdom-global": "1.7.0",
     "keycode": "2.1.1",
     "lodash": "4.11.2",
     "material-ui": "0.15.0",
-    "mocha": "2.4.5",
     "moment": "2.13.0",
     "node-sass": "3.7.0",
     "object.entries": "1.0.3",
     "ramda": "^0.22.1",
+    "raven-js": "^3.7.0",
```

Comment:
additionally, it looks like your installation of `raven-js` isn't reflected in `shrinkwrap.json`. here's what you'll probably need to do to get that to work:

```
git checkout master -- package.json
# restart docker-compose
docker-compose run watch npm prune
docker-compose run watch npm install --save raven-js
```

then check in the changes to `package.json` and `shrinkwrap.json`. That should do it!